### ISSUE TITLE
just right after completing install steps...

### ISSUE THREAD

**Issue by okliv** [BaseSecrete/rorvswild#12]
NoMethodError (undefined method `[]' for nil:NilClass):
        rorvswild (1.5.3) lib/rorvswild/plugin/action_controller.rb:17:in `start'
        activesupport (4.2.8) lib/active_support/notifications/fanout.rb:98:in `start'
        activesupport (4.2.8) lib/active_support/notifications/fanout.rb:42:in `block in start'

🤷

**Comment by alexisbernard** [comment]
What are your Rails and Ruby versions ?

**Comment by okliv** [comment]
$ ruby -v
ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-linux]

**Comment by alexisbernard** [comment]
I have fixed it. Can you confirm me it's also the case on your side by updating your Gemfile ? Then I will release a new version.

```ruby
gem "rorvswild", git: "https://github.com/BaseSecrete/rorvswild.git", ref: "db5187bf7da991d80afc3f6aef1fb9fac7baaa91
```

**Comment by okliv** [comment]
now it just gives me "We're sorry, but something went wrong." after a minute of website not responding without any errors in a log :(

**Comment by syedhassan** [comment]
@okliv try using gem 1.5.1 so in Gemfile
`gem 'rorvswild', '1.5.1'`

**Comment by okliv** [comment]
same silent freeze

**Comment by alexisbernard** [comment]
@okliv can you provide me your Gemfile ? I suspect there is a conflict with another gem.

**Comment by okliv** [comment]
unfortunately, i already done `$ bundle update` and now `ref: "db5187bf7da991d80afc3f6aef1fb9fac7baaa91` version allowed me to start my website. I do not have an old version of `Gemfile.lock`

**Comment by okliv** [comment]
implicitly, maybe the list of gems could help

```
*** LOCAL GEMS ***

actionmailer (4.2.8)
actionpack (4.2.8)
actionview (4.2.8)
activeadmin (1.4.3)
activejob (4.2.8)
activemodel (4.2.8)
activerecord (4.2.8)
activesupport (4.2.8)
addressable (2.7.0, 2.6.0)
airrecord (1.0.2)
akami (1.3.1)
anyway_config (2.0.2, 2.0.0.pre)
arbre (1.2.1)
arel (6.0.4)
babel-source (5.8.35)
babel-transpiler (0.7.0)
bcrypt (3.1.13)
bigdecimal (default: 1.4.1)
binding_of_caller (0.8.0)
builder (3.2.4, 3.2.3)
bundler (default: 1.17.2)
byebug (11.1.3, 11.0.1)
carrierwave (1.3.1)
chronic (0.10.2)
cmath (default: 1.0.0)
coffee-rails (4.1.1)
coffee-script (2.4.1)
coffee-script-source (1.12.2)
concurrent-ruby (1.1.6, 1.1.5)
connection_pool (2.2.2)
crass (1.0.6, 1.0.4)
csv (default: 3.0.9)
date (default: 2.0.0)
debug_inspector (0.0.3)
declarative (0.0.10)
declarative-option (0.1.0)
devise (4.7.1, 4.6.2)
did_you_mean (1.3.0)
e2mmap (default: 0.1.0)
enumerize (2.3.1)
erubis (2.7.0)
etc (default: 1.0.1)
execjs (2.7.0)
faraday (0.17.3, 0.15.4)
fcntl (default: 1.0.0)
ffi (1.12.2, 1.11.1)
fiddle (default: 1.0.0)
fileutils (default: 1.1.0)
font-awesome-rails (4.7.0.5)
formadmin (0.2.1)
formtastic (3.1.5)
formtastic_i18n (0.6.0)
forwardable (default: 1.2.0)
foundation-rails (6.6.2.0, 6.5.3.0)
globalid (0.4.2)
google-ads-common (1.0.2)
google-ads-savon (1.0.3)
google-adwords-api (1.5.0)
google-api-client (0.36.4)
google_drive (3.0.4)
googleauth (0.12.0, 0.10.0)
gretel (3.0.9)
gyoku (1.3.1)
has_scope (0.7.2)
httpclient (2.8.3)
httpi (2.4.4)
i18n (0.9.5)
inherited_resources (1.9.0)
io-console (default: 0.4.7)
ipaddr (default: 1.2.2)
irb (default: 1.0.0)
item (1.0.1)
jbuilder (2.9.1)
jquery-rails (4.4.0, 4.3.5)
json (default: 2.1.0, 1.8.6)
jwt (2.2.1)
kaminari (1.2.0, 1.1.1)
kaminari-actionview (1.2.0, 1.1.1)
kaminari-activerecord (1.2.0, 1.1.1)
kaminari-core (1.2.0, 1.1.1)
launchy (2.5.0, 2.4.3)
letter_opener (1.7.0)
libv8 (3.16.14.19 x86_64-linux)
logger (default: 1.3.0)
loofah (2.5.0, 2.2.3)
mail (2.7.1)
maruku (0.7.3)
matrix (default: 0.1.0)
memoist (0.16.2)
mime-types (3.3.1, 3.2.2)
mime-types-data (3.2020.0425, 3.2019.0331)
mini_magick (4.10.1, 4.9.5)
mini_mime (1.0.2)
mini_portile2 (2.4.0)
minitest (5.14.0, 5.11.3)
multi_json (1.14.1)
multipart-post (2.1.1)
mutex_m (default: 0.1.0)
mwheelintent-rails (1.3.2)
mysql2 (0.3.18)
net-http-persistent (4.0.0, 3.1.0)
net-ping (2.0.7, 2.0.6)
net-telnet (0.2.0)
nio4r (2.5.2, 2.4.0)
nokogiri (1.10.9, 1.10.3)
non-stupid-digest-assets (1.0.9)
nori (2.6.0)
openssl (default: 2.1.2)
orm_adapter (0.5.0)
os (1.1.0, 1.0.1)
ostruct (default: 0.1.0)
power_assert (1.1.3)
prime (default: 0.1.0)
psych (default: 3.1.0)
public_suffix (4.0.5, 3.1.1)
puma (4.3.3, 4.0.1)
rack (1.6.13, 1.6.11)
rack-accept (0.4.5)
rack-cors (1.0.6)
rack-test (0.6.3)
rails (4.2.8)
rails-deprecated_sanitizer (1.0.3)
rails-dom-testing (1.0.9)
rails-html-sanitizer (1.3.0, 1.0.4)
railties (4.2.8)
rake (13.0.1, 12.3.3, 12.3.2)
ransack (1.8.10)
rb-fsevent (0.10.4, 0.10.3)
rb-inotify (0.10.1, 0.10.0)
rdoc (default: 6.1.0, 4.3.0)
ref (2.0.0)
representable (3.0.4)
responders (2.4.1)
retriable (3.1.2)
rexml (default: 3.1.9)
rorvswild (1.5.3)
rss (default: 0.2.7)
ruby-next-core (0.8.0)
sass (3.7.4)
sass-listen (4.0.0)
sass-rails (5.0.7)
scanf (default: 1.0.0)
sdbm (default: 1.0.0)
sdoc (0.4.2)
shell (default: 0.7)
signet (0.14.0, 0.12.0)
sitemap_generator (6.1.0, 6.0.2)
slack-notifier (2.3.2)
socksify (1.7.1)
sprockets (3.7.2)
sprockets-es6 (0.9.2)
sprockets-rails (3.2.1)
stringio (default: 0.0.2)
strscan (default: 1.0.0)
sync (default: 0.5.0)
test-unit (3.2.9)
therubyracer (0.12.3)
thor (1.0.1, 0.20.3)
thread_safe (0.3.6)
thwait (default: 0.1.0)
tilt (2.0.10, 2.0.9)
tracer (default: 0.1.0)
turbolinks (5.2.1, 5.2.0)
turbolinks-source (5.2.0)
turnout (2.5.0)
tzinfo (1.2.7, 1.2.5)
uber (0.1.0)
uglifier (4.2.0, 4.1.20)
warden (1.2.7)
wasabi (3.5.0)
web-console (2.3.0)
webrick (default: 1.4.2)
whenever (1.0.0)
xmlrpc (0.3.0)
zlib (default: 1.0.0)
```

**Comment by alexisbernard** [comment]
> unfortunately, i've already done `$ bundle update` and now `ref: "db5187bf7da991d80afc3f6aef1fb9fac7baaa91` version allowed me to start my website. I do not have an old version of `Gemfile.lock`

Does it means it works with the ref version `db5187bf7da991d80afc3f6aef1fb9fac7baaa91` ?

I tried to setup a rails app with all the gems you listed, but I have no errors.

**Comment by okliv** [comment]
Nevertheless, the truth is that i only run `bundle update` and now it works

**Comment by alexisbernard** [comment]
Great ! I have released version 1.5.4.
